README.md

```diff
@@ -1,6 +1,27 @@
-# Jenkins Branch API Plugin
+Jenkins Branch API Plugin
+===
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/branch-api)](https://plugins.jenkins.io/branch-api)
+[![Changelog](https://img.shields.io/github/v/tag/jenkinsci/branch-api-plugin?label=changelog)](https://github.com/jenkinsci/branch-api-plugin/blob/master/CHANGELOG.md)
```

Comment:
I'm confused by this one. If you look at the file, you'll see that it shows `branch-api-2.5.2.1` when I expected it to be `2.5.4`.

Reply:
I would revisit this if it persists after this PR is merged.

Reply:
@car-roll It's because 2.5.2.1 was released more recently than 2.5.4 (I backported a change from 2.5.4 to 2.5.2.1 right after I released 2.5.4), and by default, `/tag` shows the latest tag by date. It looks like you can add `&sort=semver` to end up with 2.5.4, although I don't know if that might have some other issues since we aren't technically using semantic versioning (what does it do for plugins with versions like 1.0?).

```suggestion
[![Changelog](https://img.shields.io/github/v/tag/jenkinsci/branch-api-plugin?label=changelog&sort=semver)](https://github.com/jenkinsci/branch-api-plugin/blob/master/CHANGELOG.md)
```

Reply:
@dwnusbaum I think I'll just leave it as is then, since I'm planning to make another release once this PR is merged.  I don't want it to cause more confusion down the road.